alot/widgets/thread.py

```diff
@@ -236,15 +255,31 @@ def _get_source(self):
             self._sourcetree = TextlinesList(sourcetxt, att, att_focus)
         return self._sourcetree
 
+    def _get_content_parts(self):
```

Comment:
I think this logic should not be part of the FocusableText widget tbh.

Reply:
It isn't? :confused: It's in MessageTree.

Reply:
Oh sorry, my bad: Github folding tricked me into believing the change is in FocussableText. I take it back :)